app/server/app/routes/data.js

```diff
@@ -0,0 +1,29 @@
+const { resolve } = require("node:path");
+const { readFile } = require("node:fs/promises");
+const express = require("express");
+const axios = require("axios");
+const logger = require("../utilities/logger");
+const log = logger.logger;
+const { getActiveSchema } = require("../middleware");
+const Assessment = require("../models").Assessment;
+
+module.exports = function (app) {
+  const router = express.Router();
+
+  router.use(getActiveSchema);
+
+  // --- get assessments from database
+  router.get("/assessments", function (req, res, next) {
+    try {
+      return Assessment.schema(req.activeSchema)
+        .findAndCountAll({})
+        .then((assessments) => res.status(200).send(assessments))
+        .catch((error) => res.status(400).send("Error! " + error));
+    } catch (error) {
+      log.error("Failed to get assessments...");
+      return res.status(400).send("Error !" + error);
```

Comment:
I'm not sure if 400 is the most appropriate error code here, but I understand this is probably a placeholder for now.

Reply:
Good point. I switched to 500 status code for now, like what we use for failed `/api` requests. These may change later when we add in more advanced querying capability.